### ISSUE TITLE
Support Mechanical Turk sandbox mode

### ISSUE THREAD

**Issue by 4knahs** [aws-beam/aws-elixir#1]
Currently the endpoint prefix is [hardcoded for mturk](https://github.com/aws-beam/aws-elixir/blob/master/lib/aws/mechanical_turk.ex#L579):

```
host = get_host("mturk-requester", client)
...
defp get_host(endpoint_prefix, client) do
    if client.region == "local" do
      "localhost"
    else
      "#{endpoint_prefix}.#{client.region}.#{client.endpoint}"
    end
end
```

Since there is a [sandbox development endpoint](https://requester.mturk.com/developer/sandbox), it would be nice to support the "mturk-requester-sandbox" prefix as well.

The following lines allow passing the prefix through the options:

```
defp request(client, action, input, options) do
    prefix = Keyword.get(options, :endpoint_prefix, "mturk-requester")
    options = Keyword.delete(options, :endpoint_prefix)
    client = %{client | service: "mturk-requester"}
    host = get_host(prefix, client)
   ...
end
```

Then, one can pass the prefix like:

```
AWS.MechanicalTurk.list_h_i_ts(client, %{}, endpoint_prefix: "mturk-requester-sandbox")
```

Not sure if this is the best approach or if it would make more sense to pass this through the client though. 

The code for the prefix fix: [https://github.com/4knahs/aws-elixir/blob/mturk-sandbox/lib/aws/mechanical_turk.ex#L577](https://github.com/4knahs/aws-elixir/blob/mturk-sandbox/lib/aws/mechanical_turk.ex#L577)

**Comment by mrallen1** [comment]
I think your suggestion is a good one. If you submit a PR, I will merge it.

**Comment by mrallen1** [comment]
Solved #2
